apps/frontend/src/components/BlogCard.tsx

```
@@ -0,0 +1,74 @@
+            to={`/blog/${id}`}
+            draggable={false}
+        >
+            <img
+                src={image || undefined}
+                width={1920}
+                height={1080}
+                draggable={false}
+                // alt={image.alt}
+                onLoad={handleLoad}
+                className={`group-hover:scale-101 h-40 rounded-t-2xl bg-neutral-600 object-contain transition-transform duration-500 sm:h-[188px] ${loading && image ? 'animate-loading' : 'animate-none'} ${featured ? 'md:h-[255px] lg:h-[362px]' : 'lg:h-[270px]'}`}
+            />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)